docs/make.jl

```diff
@@ -26,8 +26,11 @@ makedocs(
         ],
         "Nonlinear Derivatives" => [
             "nonlinear_derivatives/nonlinear_diffusion.md"
-        ]
-    ]
+        ],
+        "Operator Tutorials" => [
```

Comment:
Move this to right after symbolic tutorials. Tutorials before manual and API.

Reply:
Yeah right